bundles/org.openhab.binding.opengarage/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="opengarage:opengarage">
+
+		<instruction-set targetVersion="1">
```

Comment:
You forget to add the thingTypeVersion property in the thing type opengarage:opengarage